assets/js/dataStorage.js

```diff
@@ -0,0 +1,26 @@
+// Local storage
+const nameInput = document.getElementById('fullName');
+const emailInput = document.getElementById('email-input');
+const messageInput = document.getElementById('message');
+const allInputs = document.querySelectorAll('input');
+
+const storeData = () => {
+  const inputData = {
+    inputName: nameInput.value,
+    inputEmail: emailInput.value,
+    inputText: messageInput.value,
+  };
+  localStorage.setItem('UserData', JSON.stringify(inputData));
+};
+
+allInputs.forEach((input) => {
+  input.addEventListener('input', storeData);
+});
+
+const prefillData = JSON.parse(localStorage.getItem('UserData'));
+
+if (prefillData) {
+  nameInput.value = prefillData.inputName;
+  emailInput.value = prefillData.inputEmail;
+  messageInput.value = prefillData.inputText;
+}
```

Comment:
- [ ] Good job with this attempt but there seems to be a few things to check in order to make it work effectively. The page is not persevering user's data at the moment, ad nothing happens after a `load` even happens on the windows. Kindly check this recorded [video](https://watch.screencastify.com/v/XTwTVUSBbH7YeYr7ZVyU) for a clue on how to go about it. :pray: